07-plotting-solar-data/displaying_data.ipynb

```diff
@@ -90,12 +125,22 @@
      "outputs": []
     },
     {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "But it can also be used for more complex plots:"
+     ]
+    },
+    {
      "cell_type": "code",
      "collapsed": false,
      "input": [
+      "plt.close()\n",
       "fig = plt.figure(figsize=(12, 12))\n",
-      "aiamap.plot(vmin=200, vmax=900, cmap=\"cubehelix\")\n",
+      "aiamap.plot(vmin=200, vmax=1200, cmap=\"cubehelix\")\n",
       "aiamap.draw_grid(grid_spacing=5) # Do this with no argument, then add one to demonstrate it\n",
+      "plt.xlim(-350, 150)\n",
+      "plt.ylim(-600, -100)\n",
       "plt.colorbar(orientation=\"horizontal\")\n",
       "plt.show()"
```

Comment:
I think you do, at least on my computer. Keeping it is more general anyway and I think that's better. Attendees aren't likely to be doing their research in IPython notebooks and it's worth them going through the motions. They're there to learn after all :P